packages/twenty-front/src/modules/object-record/hooks/useDeleteManyRecords.ts

```diff
@@ -67,44 +68,69 @@ export const useDeleteManyRecords = ({
         (batchIndex + 1) * mutationPageSize,
       );
 
+      const currentTimestamp = new Date().toISOString();
```

Comment:
making the useDeleteMany having the EXACT same logic as useUpdateOne but batched